quicklogic_fasm/qlassembler/pp3/ql725a.py

```diff
@@ -1,14 +1,160 @@
 #!/usr/bin/env python3
 from quicklogic_fasm.qlassembler import QLAssembler as qlasm
+import os
 
 class QL725AAssembler(qlasm.QLAssembler):
```

Comment:
Is the class capable now of disassembling a bitstream with SPI header and checksum? For now it should at least detect and discard those. In a subsequent PR we may add decoding data from there.

Reply:
I should have added disassembly capabilities in this PR. But it can also be added later. For now I included the PP3 specific functions for disassembly and handling RAM initialization that will raise `NotImplementedError`.